pkgs/applications/kde/kdegraphics-thumbnailers-fix-hang.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/ps/gscreator.cpp b/ps/gscreator.cpp
```

Comment:
Could we fetch this from Phabricator using `fetchpatch`, instead of including it in the Nixpkgs tree?